src/app/server/DataModelHandler.h

```diff
@@ -30,7 +31,7 @@
  * data model messages.
  *
  */
-void InitDataModelHandler();
+void InitDataModelHandler(chip::Messaging::ExchangeManager * exchangeMgr);
```

Comment:
I wonder whether this header should move (in a followup?) to src/app/util, which is where the .cpp lives....

Reply:
> I wonder whether this header should move (in a followup?) to src/app/util, which is where the .cpp lives....

I agree. I notice that it lives in `src/app/server` too. Will fix that in a followup.

Reply:
I end up fixing it here.